SynSSPI.pas

```diff
@@ -55,7 +55,7 @@
 {$I Synopse.inc} // define HASINLINE and other compatibility switches
 
 interface
-
+{$ifdef WIONDOWS}

```

Comment:
WINDOWS :)

Reply:
OK, it seems the same implementation as mine :)